src/Util/Crypto.php

```diff
@@ -185,8 +185,8 @@ private static function pemToRsa($key)
 
 	/**
 	 * @param string $key key
-	 * @param string $m   modulo reference
-	 * @param string $e   exponent reference
```

Comment:
Those white spaces are actually wanted, so all the parameter comments are aligned.